pkg/pxc/app/statefulset/node.go

```diff
@@ -265,6 +267,68 @@ func (c *Node) AppContainer(spec *api.PodSpec, secrets string, cr *api.PerconaXt
 		}...)
 	}
 
+	if cr.CompareVersionWith("1.19.0") >= 0 {
```

Comment:
At the end, we merely want to set this

```
			appc.Env = append(appc.Env, corev1.EnvVar{
				Name:  ldPreloadKey,
				Value: ldPreloadValue,
			})
```

So I would propose we extract the calculation of `ldPreloadValue` to a function to make everything tidier.